src/transformers/models/conditional_detr/modeling_conditional_detr.py

```diff
@@ -35,6 +35,7 @@
     requires_backends,
 )
 from ...utils.backbone_utils import load_backbone
+from ..detr.modeling_detr import DETR_ATTENTION_CLASSES
```

Comment:
We should not do this import, instead modules should be included with `# Copied from ` statement and updated with `make fix-copies`

Reply:
But won't that be redundant? I mean in this case we are just redefining the exact same code in this class whereas it can be imported easily?

Reply:
see the comment above :)